requirements.txt

```diff
@@ -28,3 +28,4 @@ six==1.15.0
 soupsieve==2.0.1
 urllib3==1.25.9
 webdriver-manager==3.1.0
+pywin32~=228
```

Comment:
So while trying to test this pull request I ran into a couple of things. 

1) The 2XX versions of `pywin32` only work in Python 2 (which is no longer [supported](https://pythonclock.org/)) :(

2) `pywin32` is a Windows only package, I would prefer to keep this _OS agnostic_

Reply:
These are good points.
I agree with you but using pywin32 was the only way I could figure out to upload files since when you click upload a windows dialog pops up.

Reply:
Have you tried the `.send_keys()` function?

It worked for me in the this sequence and for the original author.
```
for input_button in input_buttons:
	parent = input_button.find_element(By.XPATH, "..")
	sibling = parent.find_element(By.XPATH, "preceding-sibling::*")
	grandparent = sibling.find_element(By.XPATH, "..")
	for key in self.uploads.keys():
		if key in sibling.text or key in grandparent.text:
			input_button.send_keys(self.uploads[key])  # <---- THIS FUNCTION UPLOADS THE FILE
```

Reply:
Yes. The dialog box did not appear to be browser based, but rather a Microsoft Windows dialog box.
Selenium is not able to handle non browser based windows.